### ISSUE TITLE
issue on last item of a list

### ISSUE THREAD

**Issue by alf45tar** [MajicDesigns/MD_Menu#3]
wrong code

```
       if (_V.value >= size - 1 )   // index set incorrectly
          _V.value = 0;

```
right code

```
       if (_V.value >= size )   // index set incorrectly
          _V.value = 0;
```

**Comment by MajicDesigns** [comment]
Which file?

**Comment by alf45tar** [comment]
MD_Menu.cpp
MD_Menu::processList
at the end of case NAV_NULL:

**Comment by MajicDesigns** [comment]
Fixed version 2.0.2
